### PR TITLE
LUN-435: Add user created vocabulary to dictionary

### PR DESCRIPTION
**Pull request by SteffiStuffel**
This pull request belongs to an issue on our [bugtracker](https://issues.tuerantuer.org/secure/RapidBoard.jspa?rapidView=27&view=detail).
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword _LUN_.

### CODE REVIEW

File: src/hooks/useGetAllWords.ts [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+export const getAllWords = async (): Promise<VocabularyItem[]> => {
+  const lunesStandardVocabulary = loadAllVocabularyItems()
+  const userVocabulary = getUserVocabularyItems()
+  let response: VocabularyItem[] = []
+  return Promise.all([lunesStandardVocabulary, userVocabulary]).then(values => {
+    response = response.concat(values[0])
+    response = response.concat(values[1])
+    return response
+  })
```
Comment (f1sh1918) [inline src/hooks/useGetAllWords.ts:14]:
```suggestion
 return Promise.all([lunesStandardVocabulary, userVocabulary]).then(values => [...values[0],...values[1]])
```

Comment (f1sh1918) [reply src/hooks/useGetAllWords.ts:14]: you can use es6 spread operator

**Review by f1sh1918 — APPROVED** [review]
just small refactoring found. nice work. Tested on ios

File: src/hooks/useGetAllWords.ts (hunk 1 above)
Comment (SteffiStuffel) [reply src/hooks/useGetAllWords.ts]: Good point, thanks.

**Review by LeandraH — APPROVED** [review]
Nice work, tested on iOS emulator

**Pull request merged**